Gems/PhysX/Code/Editor/Source/ComponentModes/PhysXSubComponentModeBase.h

```diff
@@ -45,4 +45,15 @@ namespace PhysX
         virtual void HandleMouseInteraction(
             [[maybe_unused]] const AzToolsFramework::ViewportInteraction::MouseInteractionEvent& mouseInteraction) {};
     };
+
+    //! NullObject implementation
```

Comment:
suggestion: It might be worth mentioning this can be used as a placeholder component mode until one is fully implemented. It would just be nice to see a note of why this is needed and what it's used for.